lib/kubernetes-deploy/kubernetes_resource/replica_set.rb

```diff
@@ -40,6 +40,14 @@ def deploy_failed?
       @pods.present? && @pods.all?(&:deploy_failed?)
     end
 
+    def failure_message
+      @pods.map(&:failure_message).compact.uniq.join("\n")
```

Comment:
Does it mean that if Shopify/shopify with 500 pods fails to start, it would print logs for all of them?